common/models/transducer/alignment.py

```diff
@@ -64,3 +68,33 @@ def rnnt_alignment(source, **kwargs):
   _, alignment = tf_forward_shifted_rnnt(log_probs, targets.get_placeholder_as_batch_major(), enc_lens, dec_lens,
                                          blank_index=blank_idx, debug=False, with_alignment=True)
   return alignment  # (B, T)
+
+
+def rna_alignment_out_type(sources, **_kwargs) -> Data:
+  """ Computes the rna-alignment Data_out_type for RNA alignment
+  B: batch, T: time, U:target/labels, V: vocabulary
+  Args:
+      sources (list[LayerBase]): [output_log_prob, real_target, base:encoder]
```

Comment:
As I said, do not make it redundant.
```suggestion
      sources (list[LayerBase]):
```